.github/workflows/reuseable-workflow-tf-docs-helm-docs.yaml

```diff
@@ -0,0 +1,67 @@
+# https://github.com/terraform-docs/gh-actions
```

Comment:
I would suggest to rename the workflow. It si pretty long and will only shown abbreviated in the actions tab. Additional that is a workflow is clear due to its location, ...
Maybe  generate-tf-helm-docs.yaml